### PR TITLE
Brutus's Office Mission - Rudimentary Implementation

### LINKED ISSUE #291: Brutus mission, office AI

### LINKED ISSUE #293: Brutus mission, office logic

### LINKED ISSUE #301: Investigate using Unity Timeline for cutscenes

### PR DESCRIPTION
**Pull request by jurcaua**
What is not included in this PR:
* balancing of the level difficulty (quite difficult if you're not at max awakeness from the very beginning)
* more clear tutorialization of the throwing controls
* restart when caught by brutus

### CODE REVIEW

**Comment by jurcaua** [comment]
Wanted to make this a draft but it just automatically made it a normal PR...
Don't merge this yet please, but feel free to do a code review.

**Review by chadleytan — APPROVED** [review]
Ayeeee! Dope stuff! Excited for everything to come together!

File: Assets/Scripts/NPCs/FieldOfVision.cs [hunk 1]
```diff
@@ -218,25 +218,26 @@ EdgeInfo FindEdge(ViewCastInfo minViewCast, ViewCastInfo MaxViewCast)
     ViewCastInfo ViewCast(float globalAngle)
 	{
         Vector3 direction = DirFromAngle(globalAngle, true);
+		print(direction);
```
Comment (fnnbrr) [inline Assets/Scripts/NPCs/FieldOfVision.cs]: Can you get rid of this now that the issue is all fixed?

(I was wondering why I was only getting 15 FPS, then I saw that I had 999+ debug messages in the console.  I removed this line, and FPS went back up to ~150!)

**Review by fnnbrr — CHANGES_REQUESTED** [review]
Just remove the print debug call in FieldOfVision and you're good to go!

File: Assets/Scripts/NPCs/FieldOfVision.cs (hunk 1 above)
Comment (jurcaua) [reply Assets/Scripts/NPCs/FieldOfVision.cs]: OOF big sads there, good catch 😨

**Review by fnnbrr — APPROVED** [review]
Let er rip!

**Pull request merged**